src/java/org/apache/cassandra/config/Config.java

```diff
@@ -513,6 +513,8 @@
      */
     public volatile double range_tombstone_list_growth_factor = 1.5;
 
+    public StorageAttachedIndexOptions sai_options = new StorageAttachedIndexOptions();
```

Comment:
@mfleming Can you review this commit please. This covers the yaml configuration changes needed for SAI.

Reply:
Looks good.